db/migrate/20190410110530_remove_email_from_general_feedbacks.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveEmailFromGeneralFeedbacks < ActiveRecord::Migration[5.2]
```

Comment:
This stuff be better to be rebased into the initial migrations, rather than added as a seperate commit if we're never going to use it, just to keep the commit history and migrations tidy

Reply:
The problems with doing that is that it would require squashing a bunch of the commit history to make sure all commits refer to a working project state that can be revisited without too much pain (e.g. when bisecting). This PR went through a few different versions, hence the long commit history. I'm happy to try and squash the history a bunch to make it a clearer path to the current version if you think it's important.

Reply:
Fair enough. It's probably not worth the pain to rebase 👍